addons/mass_mailing/static/src/js/radio_info_widget.js

```diff
@@ -0,0 +1,35 @@
+odoo.define('mass_mailing.radio_info', function (require) {
+    "use strict";
+    
+    var relational_fields = require('web.relational_fields');
+    var registry = require('web.field_registry');
+    var core = require('web.core');
+    var _t = core._t;
+    
+    var FieldRadio = relational_fields.FieldRadio;
+
+    var FieldRadioInfo = FieldRadio.extend({
```

Comment:
If you really want to go that way, tests will be necessary. However adding a just just to append a span seems a bit overkill to me.